book/plan.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Is there a use for this guide, the `developers` and `assess` ones to be on master considering they are empty and still a work-in-progress before anyone can use them?

Reply:
I'm afraid, I don't understand your question. Are you asking about `plan` guide in particular, or all of the guides except `discover`?

Reply:
Yea all three (except `discover`), they all look empty and still WIP

Reply:
Removed WIP files in 121b05f